package.json

```diff
@@ -11,7 +11,7 @@
   "files": [
     "build/*.js",
     "src/**/*.js",
-    "dist/index.cjs"
+    "dist/index.umd.cjs"
```

Comment:
any reason we're shipping a [umd](https://github.com/umdjs/umd) file?

Reply:
It was a UMD file [pre-vite](https://github.com/netlify/framework-info/pull/819/files#diff-ac151ec034da625904b09e1b25d82650d5f0d34fb9ae7f0782ebcb19e7a3c7bdL19).

And we target this is because of the browser usage of this library (where we detect the framework during the 'import project' part of adding/creating a new site in the Netlify UI)